### PR TITLE
Further Android detection via elf and symbols.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: libr/bin/p/bin_elf.inc [hunk 1]
```diff
@@ -966,6 +966,13 @@ static void lookup_symbols(RBinFile *bf, RBinInfo *ret) {
 				is_rust = true;
 				ret->lang = "rust";
 			}
+			if (!is_android && strstr (symbol->name, "android_")) {
```
Comment (Maijin) [inline libr/bin/p/bin_elf.inc]: Not a fan of the strstr that's too generic imho

Comment (devnexen) [reply libr/bin/p/bin_elf.inc]: That s not false ... it is just a little improvement for a difficult recognition :-) I wish Android had its own elf osabi :-)

Comment (Maijin) [reply libr/bin/p/bin_elf.inc]: I'm not saying it's false, I'm saying that's too generic, the chance of getting a symbol containing  "android_" on a non-android executable is fairly high

**Comment by devnexen** [comment]
After reconsidering, closing ...

**Pull request closed without merging**